tests/storybook.ts

```diff
@@ -1,12 +1,22 @@
-import { runInRepo } from '../utils.ts'
+import { $, runInRepo } from '../utils.ts'
 import type { RunOptions } from '../types.d.ts'
+import { getCommand } from '@antfu/ni'
 
 export async function test(options: RunOptions) {
 	await runInRepo({
 		...options,
-		repo: 'storybookjs/builder-vite',
-		branch: 'main',
-		build: 'prepublish',
-		test: ['build-examples', 'test-ci'],
+		repo: 'storybookjs/sandboxes',
+		dir: 'next',
+		testSubdirectory: 'react-vite/default-ts/after-storybook',
```

Comment:
is there a reason to pick this specific subdirectory, i suppose there are multiple setups with vite in storybooks tests. Is this the one with the biggest test coverage or quickest runtime? I'm totally fine with making a compromise here to reduce execution time. But if other subdirectories would increase coverage, is there a way to add them?

Regarding the testSubdirectory, could this be prevented as well with using `pnpm --dir <testSubDirectory>` calls in the script block directly? or in the `"scripts"` section of the storbookjs/sandboxes monorepo root?

Reply:
We have different sandboxes testing renderer/builder and TypeScript/JavaScript combinations. These are all our sandboxes, which we have in place using the Vite builder:

- react-vite/default-js
- react-vite/default-ts
- react-vite/prerelease-ts
- solid-vite/default-js
- solid-vite/default-ts
- vue3-vite/default-js
- vue3-vite/default-ts
- html-vite/default-js
- html-vite/default-ts
- svelte-vite/default-js
- svelte-vite/default-ts
- svelte-kit/skeleton-js
- svelte-kit/skeleton-ts
- svelte-kit/prerelease-ts
- lit-vite/default-js
- lit-vite/default-ts
- preact-vite/default-js
- preact-vite/default-ts